bin/srv.py

```diff
@@ -225,7 +225,7 @@ def wttr(location = None):
     user_agent = request.headers.get('User-Agent').lower()
 
     html_output = True
-    if 'curl' in user_agent or 'wget' in user_agent:
+    if 'curl' in user_agent or 'wget' in user_agent or 'httpie' in user_agent:
```

Comment:
Maybe it's better to do it like this?
```python
if any(
    agent in user_agent
    for agent in ["curl", "wget", "httpie"]
    ):
    html_output = False
```

I think that is more scaleable way to do that logic.